pkg/server/config.go

```diff
@@ -36,6 +36,16 @@ func NewConfig() *Config {
 		}
 	}
 
+	if disableGoogleAuth == "" {
+		// Will be enabled by default
```

Comment:
Lets keep disabled by default just like GitHub?

Reply:
The default behaviour is enabled by default for both, do you want me to change it to disabled by default for both because it's meaningless until a valid client_id, client_secret & redirect_uri are specified.

Reply:
I think it is disabled by default for GitHub.
Yes, it should be disabled by default for both.